src/resources/Alteration/C.ts

```diff
@@ -0,0 +1,55 @@
+import { CorrectNameOptions, FilingCodes } from '@/enums/'
+import { CorpTypeCd, GetCorpFullDescription } from '@bcrs-shared-components/corp-type-module/'
+import { ResourceIF } from '@/interfaces/'
+
+export const AlterationResourceC: ResourceIF = {
+  entityReference: 'Company',
+  contactLabel: 'Registered Office',
+  displayName: GetCorpFullDescription(CorpTypeCd.CONTINUE_IN),
+  addressLabel: 'Registered Office',
+  filingData: {
+    filingTypeCode: FilingCodes.ALTERATION,
+    entityType: CorpTypeCd.CONTINUE_IN,
+    priority: false
+  },
+  // Conditionally used in place of filingData when switching from BC (or C) to ULC:
+  additionalFilingData: {
+    filingTypeCode: FilingCodes.ALTERATION_BC_TO_ULC,
+    entityType: CorpTypeCd.BC_COMPANY,
```

Comment:
This is not an error. This additional filing data uses entity type BC to look up the filing code/fee.

(It's this way in AlterationResourceBc also.)